src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -15,7 +16,7 @@ import { artworkConnection } from "../artwork"
 const standardSectionFields: GraphQLFieldConfigMap<any, ResolverContext> = {
   ...InternalIDFields,
   component: {
-    type: HomeViewComponent,
+    type: new GraphQLNonNull(HomeViewComponent),
```

Comment:
🙌 
If there's any other similar prop we're sure to receive, we should wrap them with it too!
I consider not doing it a blocker on MP PRs lately 😂